test/spec/runners/epoch_processing.ex

```diff
@@ -55,7 +57,12 @@ defmodule EpochProcessingTestRunner do
     # handle_case(testcase.handler, pre, post)
   end
 
-  # def handle_case("eth1_data_reset", pre, post) do
-  #   assert process_eth1_data_reset(process_eth1pre.eth1_data) == post.eth1_data
-  # end
+  def handle_case("eth1_data_reset", pre, post) do
+    result = EpochProcessing.process_eth1_data_reset(pre)
+
+    case post do
+      nil -> elem(result, 0) == :error
```

Comment:
you are not asserting anything in this line

Reply:
Corrected that. Thanks

Reply:
I think the issue with the build from dialyzer is that we are checking for the error but it is not returned explicitly in the function.

What if I rescue it and reraise it?

Reply:
Ah, gotcha. Is there a spec test where post is actually `nil`? Otherwise you just need to handle the `:ok` scenario

Reply:
It seems this function always succeeds so we can update the @spec removing the `:error` possibility. That was my bad, sorry about that

Reply:
Yeah, it never fails